deisctl/units/deis-mesos-marathon.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Mesosphere Marathon
+After=docker.service
+Requires=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+Restart=on-failure
+RestartSec=20
+TimeoutStartSec=0
+ExecStartPre=-/bin/sh -c "etcdctl get /deis/scheduler/mesos/marathon >/dev/null 2>&1 || etcdctl mk /deis/scheduler/mesos/marathon"
+ExecStartPre=/bin/sh -c "etcdctl set /deis/scheduler/mesos/marathon $COREOS_PRIVATE_IPV4"
+ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/mesos-marathon` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
+ExecStartPre=-/usr/bin/docker kill deis-mesos-marathon
+ExecStartPre=-/usr/bin/docker rm deis-mesos-marathon
+ExecStart=/usr/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/mesos-marathon` && docker run \
```

Comment:
Could this be a one-liner? I know it's long, but we don't break onto multiple lines in other unit files.